### PR TITLE
update database and table materialization

### PR DESCRIPTION
**Pull request by bachng2017**
- enhance database materialization to work when databases exists
- update table materialization to work with latest mindsdb (stop using predictor_name)

### CODE REVIEW

**Comment by bachng2017** [comment]
@ZoranPandovski  pls review above MR when you are available. Thanks

File: dbt/include/mindsdb/macros/materialization/table.sql [hunk 1]
```diff
@@ -14,17 +13,18 @@
...
+  -- final
   {% set target_relation = target_relation_list | join('.') %}
 
   -- ... setup database ...
   -- ... run pre-hooks...
 
   -- build model
   {% call statement('main') %}
-       {{ apply_predictor_wrap(sql, predictor_name, target_relation) }}
+       {{ apply_predictor_wrap(sql, target_relation) }}
```
Comment (ea-rus) [inline dbt/include/mindsdb/macros/materialization/table.sql:27]: Maybe it is worth to rename function to save_to_table_wrap

Comment (bachng2017) [reply dbt/include/mindsdb/macros/materialization/table.sql:27]: no problem, will do that

Comment (bachng2017) [reply dbt/include/mindsdb/macros/materialization/table.sql]: done

**Review by ea-rus — APPROVED** [review]

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**